### DISCUSSION TITLE
Is there an official way to do routing?

### DISCUSSION THREAD

**Discussion by tjbo** [BuckleTypes/bs-react-native#759] [Q&A] [answered]
I looked through the APIs and Components, but couldn't see anything.

Was wondering if there was a document on this, (or maybe it just hasn't been added yet).

Enjoying this framework so far!

Thanks;
Tom

**Comment by MoOx** [comment, marked as answer]
You should have a look to react-navigation, the easiest way to get started: https://reactnavigation.org

Bindings are available: https://github.com/rescript-react-native/rescript-react-navigation

**Reply by tjbo** [reply]
Great thanks.
